psycop/common/model_training_v2/artifact_savers/task_disk_saver.py

```diff
@@ -0,0 +1,28 @@
+import pickle as pkl
+from pathlib import Path
+
+from psycop.common.model_training_v2.artifact_savers.base_artifact_saver import (
+    BaselineArtifactSaver,
+)
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.base_trainer import (
+    BaselineTrainer,
+    TrainingResult,
+)
+
+
+@BaselineRegistry.artifact_savers.register("task_disk_saver")
+class TaskDiskSaver(BaselineArtifactSaver):
+    def __init__(self, experiment_path: str):
```

Comment:
Perhaps this should signify which artifacts it is capable of saving, e.g. using arguments to init? E.g. `save_trainer`, `save_metric` etc.? Is there anything else in TrainingResult we might save?